docs/app/pages/components/components.module.ts

```diff
@@ -13,7 +13,7 @@ const ROUTES: Routes = [
             { path: 'component-list', loadChildren: () => import('./components-sections/component-list/component-list.module').then(m => m.ComponentsListModule) },
             { path: 'color', loadChildren: () => import('./components-sections/color/color.module').then(m => m.ComponentsColorModule) },
             { path: 'conduits', loadChildren: () => import('./components-sections/conduits/conduits.module').then(m => m.ComponentsConduitsModule) },
-            { path: 'dashboard', loadChildren: () => import('./components-sections/dashboard/dashboard.module').then(m => m.ComponentsDashboardModule) },
```

Comment:
Is this change correct?

Reply:
It reflects the current module file name for the dashboard section with both the dashboard and widgets component. We can rename it back if required.

Reply:
Yes, please rename it back so that it matches the name of the documentation category that it represents.